src/main/java/uk/gov/hmcts/reform/coh/controller/DecisionController.java

```diff
@@ -67,7 +70,7 @@ public ResponseEntity createDecision(@PathVariable UUID onlineHearingId, @Reques
             return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Online hearing not found");
         }
 
-        ValidationResult result = DecisionRequestValidator.validate(request);
```

Comment:
Instead of the DecisionRequestValidator being responsible for executing the validation of each field. That task is handed off to the Validation class and DecisionRequestValidator then just becomes a holder of the validation strategies and Validation is the thing that will execute each strategy